sigs/runtime-charter.md

```diff
@@ -0,0 +1,150 @@
+## CNCF Runtime SIG Charter
+
+
+### Primary Author: Quinton Hoole
+
+2019-09-02
+
+Also reviewed and contributed to by:
+
+*   Liz Rice
+*   Brian Grant
+
+## Introduction
+
+This is the charter referred to in “[CNCF
+SIGs](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#sig-charter)”
+by the CNCF TOC, and consistent with the [proposed SIG
+definition](https://github.com/cncf/toc/blob/master/sigs/proposed.md).
+
+
+## Areas Considered In Scope
+
+Workload execution and management systems, components and interfaces
+used in [modern cloud-native
+environments](https://github.com/cncf/toc/blob/master/DEFINITION.md)
+are in scope, including:
+
+
+
+*   generalized orchestration, autoscaling, scheduling, execution, container
+    runtimes, sandboxing, virtualization, image packaging and distribution and specialized architectures thereof, e.g. 
```

Comment:
"image packaging and distribution" ->  "image packaging/distribution" ( the 2 `and`(s) read odd).